_data/communication.yml

```
@@ -6,6 +6,13 @@ websites:
       - totp
     doc: https://help.actionnetwork.org/hc/en-us/articles/217206826
 
+  - name: Airship
+    url: https://www.airship.com/
+    img: airship.png
+    tfa:
+      - totp
+    doc: https://docs.airship.com/tutorials/security/2fa/
+
```

```suggestion
    exception: "2FA is only available upon request for customers on their Essential and Comprehensive plans."
```
As per [Preparing for Two Factor Authentication](https://support.airship.com/hc/en-us/articles/115001994786-Preparing-for-Two-Factor-Authentication).